jenkins/automation/vra_infra_provisioner.groovy

```diff
@@ -103,8 +104,12 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
                 sh label: '', script: '''
                     pushd solutions/vmware/terraform/
+                        echo "VM list is empty" > vm_details.txt
                         terraform apply --auto-approve
-                        if [ "$?" -ne 0 ]; then echo -e '\nERROR: Infra Provision Failed!!\n'; else echo -e '\n---SUCCESS---\n'; fi
+                        if [ "$?" -ne 0 ]; then echo -e \'\\nERROR: Infra Provision Failed!!\\n\'; else echo -e \'\\n---SUCCESS---\\n\'; fi
+                        VM_HOSTNAMES=$(terraform show -json terraform.tfstate | jq .values.root_module.resources[].values.resources | jq -c \'[ .[] | select( .type | contains("Cloud.vSphere.Machine")) ]\' | tr \',\' \'\\n\' | awk -F \'\\\' \'/cloudConfig/ { print $8}\' | sed \'s/nfqdn: //g\')
```

Comment:
can we add comma in values of list of VM's?

Reply:
Fixed and tested with  https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/sv_space/job/provision-vm-debug/11/console

![image](https://user-images.githubusercontent.com/63289460/212626887-f8729342-f43b-4ffa-a6a4-4711f5f3f1a0.png)